.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+  # Autoformat: Docker files
+  - repo: local
+    hooks:
+      - id: hadolint
+        name: hadolint
+        language: docker_image
+        types:
+          - dockerfile
+        entry: hadolint/hadolint:v2.8.0-alpine hadolint ./docker/Dockerfile-*
```

Comment:
same :)

Reply:
Unfortunately, nope, the Dockerfiles we have in the repo don't pass the standards

<img width="1786" alt="Screenshot 2022-05-19 at 2 21 30 PM" src="https://user-images.githubusercontent.com/19776877/169253652-7ea5670e-42a2-4f3d-8100-9a3705080110.png">

I haven't touched that in this PR cause I was thinking of creating a new PR with those fixes

Reply:
as for me it's fine to do it in this PR as well, but as you wish. also we should add a `pre-commit run --all-files` in CI, instead of the format.sh (either in this PR if you want to adapt to hadolint, or in the PR where that's done).

Reply:
I'll do the hadolint check in another PR so that we keep things simple by following the separation of concern principle :)